gitlab-pipeline/internal-variables.yml

```diff
@@ -0,0 +1,15 @@
+variables:
+  # Internal address for nfs sstate cache server (northamerica-northeast1-b)
+  SSTATE_CACHE_INTRNL_ADDR: "10.162.0.25"
```

Comment:
why are we hardcocing the IP? cant we use the staticcally allocated ip, or a hostname?

Reply:
I honestly don't know.

Reply:
Might be taking off on a tangent here, but why are we even using `dind` as opposed to running straight on the `VM`, and then mounting `/dev` etc in the container?

Reply:
(copying from chat in Slack)

@alfrunes Setting in in the gitlab-runner configuration is also a good idea! Thanks.

One benefit of having it in the code is that allow us for trying things out while migrating to a new server (for example some weeks ago when I had to recreate the machine). Another benefit is, well, that it is better documented (at least until we have terraform for this machine).

So I think I'll leave it as it is with this PR.

Reply:
> Might be taking off on a tangent here, but why are we even using `dind` as opposed to running straight on the `VM`, and then mounting `/dev` etc in the container?

@oleorhagen Quite a tangent, yes! This is a big topic that I would like to understand more. It has been challenged already by @0lmi and the truth is that both @alfrunes and me do not know _exactly_ why we had to do it this way (we both try the proper way before). So do you care to bring it up in the next QA meeting? :memo:

Reply:
Done :heavy_check_mark: